src/main/java/org/dependencytrack/notification/publisher/SendMailPublisher.java

```diff
@@ -141,7 +143,7 @@ private void sendNotification(final PublishContext ctx, Notification notificatio
                     .from(smtpFrom)
                     .to(destinations)
                     .subject(emailSubjectPrefix + " " + notification.getTitle())
-                    .body(content)
+                    .body(StringEscapeUtils.unescapeHtml4(content))
```

Comment:
I wonder if this will cause issues when the MIME type is `text/html` or similar, which would prompt mail clients to attempt to render the HTML. I'm not sure we can guarantee that the notification content is save to render, or it even makes sense to do so. For example, vulnerability descriptions might include HTML as code snippets.

Reply:
So the simplest solution would be to either unescape it if its text/plain and then its text/html leave it.

The thing is I saw is that the content is not perfectly suitable for rendering in hml either, if its left, as it was. & symbol is rightly escaped to &amp;, but the § or ' doesn't get escaped, and therefore can't be rendered in html. For that would the simplet solution be unescape it and escape it again, cause StringEscapeUtils.escapeHtml4, does it the right way. Or do you know a way that the content isn't escaped in the first place all along.